style.css

```diff
@@ -0,0 +1,273 @@
+* {
+    font-family: sans-serif;
+}
+body {
+  margin: 0;
+}
+.grid {
+  grid-template-columns: 300px 1fr 400px;
+  grid-template-rows: 100vh;
+}
+.left-bar,
+.right-bar,
+header,
+.text,
+.shadow-gap,
+.content {
+  border: solid 1px rgb(172, 172, 172);
+}
+.g, .grid {
+  display: grid;
+}
+.shadow-gap {
+    background-color: rgb(208, 208, 208);
+}
```

Comment:
I haven't used rgb for colors, this is interesting! I'd like to understand better how to set the values to colors I want

Reply:
when you write any color like white, you get the white box, if you hover over that color you will get the color pallet, you can adjust which color you want and it will automatically adjust the rgb for you. you can also use hex codes, so if you want a specific color you can google for the hex code and write it in